### ISSUE TITLE
Each port value must be unique

### ISSUE THREAD

**Issue by MikeiLL** [WordPress/gutenberg#49620]
### Description

When adding port specification to `.wp-env.json`:
```
  "port": 8890,
```

Returns error from https://github.com/WordPress/gutenberg/blob/b7c693b57df1cb89ec2f15ef25cfd2a6d816ecf3/packages/env/lib/config/config.js#L163

Seems I need to also include:

```
  "testsPort": 8891,
```

This is the entire config file:

```
{
  "core": null,
  "port": 8890,
  "testsPort": 8891,
  "config": {
    "WP_ENVIRONMENT_TYPE": "development",
    "DOCKER_REQUEST_URL": "http://example.test",
    "WP_SITEURL": "http://example.test",
    "WP_HOME": "http://example.test"
  },
  "mappings": {
    "wp-content/plugins/my-plugin": "./my-plugin",
    "/bin": "./bin",
    "bootstrap.php": "./bootstrap.php",
    "phpunit.xml.dist": "./phpunit.xml.dist",
    "ssl": "./ssl",
    "/etc/apache2/sites-available/000-default.conf": "./ssl/000-default.conf",
    "/etc/apache2/mods-available/ssl.load": "./ssl/ssl.load"
  }
}
```

Is this something that should be noted in the docs?

Thanks.

### Step-by-step reproduction instructions

Set the above noted config file and run `wp-env start`.

### Screenshots, screen recording, code snippet

// see above, please.

### Environment info

Darwin/Mac/OS X
Latest version of wp-env.

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by noahtallen** [comment]
You can also work around this issue like so:

```json
{
	"env": {
		"development": {
			"port": 8890
		}
	}
}
```

But you're definitely right, the documentation doesn't match how it works.

**Comment by noahtallen** [comment]
Fixed in #50071
